rmpv/src/lib.rs

```diff
@@ -30,6 +30,19 @@ enum IntPriv {
     NegInt(i64),
 }
 
+/// Name of Serde newtype struct to Represent Msgpack's Ext
+/// Msgpack Ext: Ext(tag, binary)
+/// Serde data model: _ExtStruct((tag, binary))
+/// Example Serde impl for custom type:
+///
+/// #[derive(Debug, PartialEq, Serialize, Deserialize)]
```

Comment:
Same thing as the above comment - just to get this to display correctly, we should surround the code in

   \```ignore
   ... code ...
   \```